diff --git a/Vagrantfile b/Vagrantfile
index 05b502ef0..15364a5dd 100644
--- a/Vagrantfile
+++ b/Vagrantfile
@@ -222,6 +222,7 @@ Vagrant.configure("2") do |config|
       if ["ubuntu2004", "ubuntu2204"].include? $os
         node.vm.provision "shell", inline: "rm -f /etc/modprobe.d/local.conf"
         node.vm.provision "shell", inline: "sed -i '/net.ipv6.conf.all.disable_ipv6/d' /etc/sysctl.d/99-sysctl.conf /etc/sysctl.conf"
+        node.vm.provision "shell", path: "../vagrant/common.sh"
       end
 
       # Disable firewalld on oraclelinux/redhat vms
@@ -274,6 +275,7 @@ Vagrant.configure("2") do |config|
             "kube_node" => ["#{$instance_name_prefix}-[1:#{$kube_node_instances}]"],
             "k8s_cluster:children" => ["kube_control_plane", "kube_node"],
           }
+          ansible.compatibility_mode = "2.0"
         end
       end
 
